container/root/app/public/index.php

```diff
@@ -8,3 +8,16 @@
 fwrite( $stderr, 'Using STDERR pipe for output' );
 
 phpinfo();
+
+// Use the below form to test uploads
+?>
+
+<form method="post" action="/" enctype="multipart/form-data">
```

Comment:
adding a quick and dirty self-posting form to allow basic-level testing that POST uploads to the server work :)